server/logging_test.go

```diff
@@ -0,0 +1,149 @@
+package server
+
+import (
+	"encoding/json"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+)
+
+type logOptionsTest map[string]string
+
+func TestEscapeDoubleQuotesOK(t *testing.T) {
+	assert.Equal(t, "test", escapeDoubleQuotes("test"))
+	assert.Equal(t, "\\\"test\\\"", escapeDoubleQuotes("\"test\""))
+	assert.Equal(t, "\\\"\\\"test\\\"", escapeDoubleQuotes("\"\"test\""))
+	assert.Equal(t, "\\\"\\\"\\\"\\\"", escapeDoubleQuotes("\"\"\"\""))
+}
+
+func TestDefaultLoggingConsoleLogConfigOK(t *testing.T) {
+	var mapCfgEscaped mlog.LoggerConfiguration
+	err := json.Unmarshal([]byte(defaultLoggingConsoleLogConfig()), &mapCfgEscaped)
+	assert.NoError(t, err)
+
+	var logOptionsConsole logOptionsTest
+
+	targetCfg := mapCfgEscaped["def"]
+
+	err = json.Unmarshal(targetCfg.Options, &logOptionsConsole)
+
+	assert.NoError(t, err)
+	assert.Equal(t, "console", targetCfg.Type)
+	assert.Equal(t, "plain", targetCfg.Format)
+	assert.Equal(t, "stdout", logOptionsConsole["out"])
+}
+
+func TestDefaultLoggingFileLogConfigOK(t *testing.T) {
```

Comment:
Can you reorganize the tests to use this pattern?
```
func TestFunctionName(t *testing.T) {
    t.Run("test description", func(t *testing.T) {
        ...
    })
    t.Run("test description", func(t *testing.T) {
        ...
    })
    ...
}
```

That way it will be more clear what you are testing in each part and what you are trying to accomplish with each test.